docs/about-filecoin/ipfs-and-filecoin.md

```diff
@@ -11,32 +11,36 @@ This page aims to explain the relationship between the IPFS and Filecoin project
 
-IPFS does not include a built-in mechanism to incentivize the storage of data for _other_ people. This is the challenge Filecoin hopes to solve. The Filecoin network creates a distributed storage marketplace for long-term storage. Nodes with a large storage capacity can rent storage to users with and get paid for it.
+IPFS alone does not include a built-in mechanism to incentivize the storage of data for _other_ people. This is the challenge Filecoin aims to solve. Filecoin is built on IPFS to create a distributed storage marketplace for long-term storage. Nodes with a large storage capacity can rent their storage to users and get paid for it.
 
-The Filecoin network ensures that data is safely stored. However, the processes of storing (sealing), verifying (proving) and unsealing (for retrieval) are computationally expensive and can take time. This is specially relevant for the retrieval of data, which should happen as fast as possible. For this reason, Filecoin enables an additional retrieval market where dedicated nodes can help quickly deliver content from the network for a payment by keeping unsealed, cached copies. This delivery mechanism may make use of IPFS.
+The Filecoin network ensures that data is safely stored. However, the processes of storing (sealing), verifying (proving) and unsealing (for retrieval) are computationally expensive and can take time. This is specially relevant for the retrieval of data, which should happen as fast as possible. For this reason, Filecoin enables an additional retrieval market where dedicated nodes can help quickly deliver content from the network for a payment by keeping unsealed, cached copies. This delivery mechanism may make use of IPFS, but is still in design (see [ResNetLab's Decentralized Data Delivery Market research](https://github.com/protocol/ResNetLab/blob/master/OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md)).
 
-Filecoin can be seen as a _cold_ storage layer, perfect to safely store large batches of data. IPFS would be the _hot_ storage layer, designed for the quick retrieval and distribution of content.
+Filecoin aims to add longer term persistence to safely store large batches of data, while IPFS optimizes for the quick retrieval and distribution of content.
 
-## Which system should I use?
+## Using IPFS and Filecoin
 
-### Using IPFS
+### Content addressing with IPFS
 
-- Data provided by user's own nodes. Otherwise must rely on other peers to voluntarily/altruistically storing data or on a centralized pinning service.
+IPFS is great for getting started using content addressing for all sorts of distributed web applications. In the majority of these cases: 
+
+- Data is provided by user's own nodes. Otherwise must rely on other peers to voluntarily/altruistically storing data or on a centralized pinning service.
```

Comment:
```suggestion
- Data is provided by the user's own nodes. Otherwise, must rely on other peers to voluntarily/altruistically storing data or on a centralized pinning service.
```